### PR TITLE
User resource template for export review vm

### LINKED ISSUE #2602: User resource template for Export Review VM
## Description

As an Airlock Manager,
I want to have a user resource template
So that I can deploy it into Research Workspace and use user resources (VMs in Guacamole) to review export data requests

## Requirement

When deployed into a Research Workspace (i.e. for reviewing export requests), a Review VM must be isolated from other resources in the workspace, as in some cases, reviewer is not allowed to access the research, they can only access the data that is intended to be exported. 

To achieve this, we can have a private endpoint for the export in progress storage account, and an NSG that allows the VM to access that storage account, but limits access for everything else.

### PR DESCRIPTION
**Pull request by tanya-borisova**
# Resolves #2602 

## What is being addressed

Created a user resource template for Export review VM

## How is this addressed

- Added a Network Security group to the windowsvm template
- Added rules denying outbound traffic from the review VM
- Added rules allowing connection to in-review export storage account
- Tested in an environment that can connect to storage account
- Tested that can't connect to other VMs from review VM (via telnet, same test on research VMs succeeds)

### CODE REVIEW

File: templates/workspace_services/guacamole/user_resources/guacamole-azure-export-reviewvm/porter.yaml [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+      description: "Delete the Extensions from the Terraform state manually"
+      command: ./delete_vm_extensions.sh
+      arguments:
+        - "{{ bundle.parameters.tfstate_resource_group_name }}"
+        - "{{ bundle.parameters.tfstate_storage_account_name }}"
+        - "{{ bundle.parameters.tfstate_container_name }}"
+        - "{{ bundle.parameters.id }}"
+
+  - terraform:
+      description: "Delete the Guacamole User Resource Service"
```
Comment (martinpeck) [inline templates/workspace_services/guacamole/user_resources/guacamole-azure-export-reviewvm/porter.yaml:137]: you've added `(Reviewer VM)` to the other descriptions
did you want to add it here too?

**Review by martinpeck — APPROVED** [review]
one small comment

File: templates/workspace_services/guacamole/user_resources/guacamole-azure-export-reviewvm/porter.yaml (hunk 1 above)
Comment (tanya-borisova) [reply templates/workspace_services/guacamole/user_resources/guacamole-azure-export-reviewvm/porter.yaml:137]: Good catch

**Comment by tanya-borisova** [comment]
/test-force-approve

No tests on this as it's a new template

**Pull request merged**